docs/timelines/running-and-editing-timelines.md

```diff
@@ -0,0 +1,138 @@
+
+- If you are using internal timecode you can set the timecode to run automatically when you raise the fader, and/or to stop when you lower the fader. See [Timeline Options - Fader](../timelines/timeline-options.md#fader-tab).
+
+- You can set the timeline to loop (when using internal timecode), see [Timeline Options - Timecode](../timelines/timeline-options.md#timecode-tab).
+
+- If you start the timecode at a point part way through the timeline, any active triggers will fire, but the state of LTP attributes on fixtures might be different because previous triggers have not fired in the usual sequence - this might result in a different look to what you programmed. You can use the [Timeline Options - Release](../timelines/timeline-options.md#release-tab) settings to help make this more predictable.
+
+### Testing a timeline
+
+You can test a timeline without running the timecode source by setting the **timecode link** button to **Unlinked**. You can then use the Play/Pause/Rewind buttons
+in the Timeline View window to run the timeline sequence. This can be useful when you don't have control of the timecode source.                  
+
+Timeline Editing
+--------------------------
+ 
+### Selecting triggers
+
+To edit a trigger, you need to select it. There are several different ways to do this.
+-   Tap on a trigger block to select all the trigger points within the block. You will see selection 
+    handles appear.  Tap again on a selected trigger block to step through the
+    trigger points within the block. 
+
+![Timeline trigger handles](/docs/images/Timeline-Handles.png)
+
+- Tap on more triggers to add them to the selection.
+
+- Draw a selection marquee box over the trigger points you want to select. Only the trigger points enclosed by the selection box will be selected, so if you want to select all points in a block they all need to be inside the box. This is a quick way to select one trigger point without having to "multi-tap" on a block.
+
+- To clear all selection use the context menu button \{Clear Trigger Selection\} or tap in an empty area of the Timeline View.
+
+### Using arrow buttons to move trigger selection
+
+The left/right arrow buttons for the track will move your selection to the previous trigger point or the next trigger point.
+
+- Left arrow - moves selection to the previous trigger point on the track.
+- Right arrow - moves selection to the next trigger point on the track.
+
+If nothing is selected, the right arrow will select the first trigger on the track and the left arrow will select the last.
+
+### Using the wheels to edit triggers
+
+When you have one or more trigger points selected, the wheels control the time, level and fade of those triggers (some types of trigger do not have all these parameters).
+
+- Wheel A sets the time.
+- Wheel B sets the level.
+- Wheel C sets the fade.
+
+You can change multiple triggers at the same time by selecting them all and using the wheels.
+
+### Using the overview bar
```

Comment:
Is this not somewhat duplicated from `docs/timelines.md`? Might be good to consolidate

Reply:
The original section introduces the basics of the overview bar - this section has more detail. Personally I think it is worth having both sections for the skim reader and those who want more.